keyboards/keychron/q1/readme.md

```diff
@@ -17,5 +17,6 @@ Flashing example for this keyboard:
     make keychron/q1/rev_0100:default:flash
 
 **Reset Key**: Hold down the key located at *K00*, commonly programmed as *Esc* while plugging in the keyboard.
+**Reset Key on Stock Firmware**: Remove the Spacebar keycap, then hold the small reset button while plugging in the keyboard.
```

Comment:
```suggestion
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button underneath the space bar
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```

Reply:
Thanks for the suggestion, but I'm not sure why you say the physical reset button should be 'briefly' pressed. IME it needed to be held during keyboard startup, same as the ESC key.

Reply:
This is just what the readme template has. If the process is different, adjust the wording accordingly.